src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -57,6 +57,7 @@ public class Invoice : StripeEntity<Invoice>, IHasId, IHasMetadata, IHasObject
         /// customer. When sending an invoice, Stripe will email your customer an invoice with
         /// payment instructions.
         /// </summary>
+        [Obsolete("Use CollectionMethod")]
```

Comment:
I would recommend changing the documentation above too.

Reply:
Added.